tests/Wallabag/ApiBundle/Controller/EntryRestControllerTest.php

```diff
@@ -497,6 +499,7 @@ public function testPostArchivedAndStarredEntry()
         $this->assertSame('http://www.lemonde.fr/idees/article/2016/02/08/preserver-la-liberte-d-expression-sur-les-reseaux-sociaux_4861503_3232.html', $content['url']);
         $this->assertSame(1, $content['is_archived']);
         $this->assertSame(1, $content['is_starred']);
+        $this->assertGreaterThanOrEqual(new \DateTime($content['starred_at']), $now);
```

Comment:
This should be in the opposite way: first the value you expect and then the expected value